kola/README.md

```diff
@@ -0,0 +1,192 @@
+podman.network                                  [all]                                   [all]   [rhcos]
+podman.noop                                     [all]                                   [all]   [rhcos]
+podman.workflow                                 [all]                                   [all]   [rhcos]
+# Run your test and see what happens
+$ sudo ./bin/kola run -b rhcos --qemu-image `pwd`/rhcos-410.8.20190502.0-qemu.qcow2 podman.noop
+=== RUN   podman.noop
+--- PASS: podman.noop (21.08s)
+PASS, output in _kola_temp/qemu-2019-05-08-1535-16606
+# git add/commit/push...
+# Open PR to get the test added!
+```
+
+## Grouping Tests
+
+Sometimes it makes sense to group tests together under a specific package, especially when these tests are related and require the same test parameters. For `kola` it only takes a forwarding function to do testing groups. This forwarding function should take `cluster.TestCluster` as it's only input, and execute running other tests with `cluster.TestCluster.Run()`.
+
+Continuing with the look at the `podman` package we can see that `podman.base` is registered like so:
+
+```golang
+    register.Register(&register.Test{
+            Run:         podmanBaseTest,
+            ClusterSize: 1,
+            Name:        `podman.base`,
+            Distros:     []string{"rhcos"},
+    })
+```
+If we look at `podmanBaseTest` it becomes very obvious that it's not a test of it's own, but a group of tests.
+
+```go
+func podmanBaseTest(c cluster.TestCluster) {
+        c.Run("info", podmanInfo)
+        c.Run("resources", podmanResources)
+        c.Run("network", podmanNetworksReliably)
+}
+```
+
+
+## Adding New Packages
+
+If you need to add a new testing package there are few steps that must be done.
+
+1. Create a new directory in `kola/tests/` which is descriptive of what will be tested.
+2. Add at least one file in the new directory with it's package the same name as it's directory name
+3. Edit the kola/registry/registry.go file to include your new package
+4. Add and register your new tests
+
+As an example, let's say you want to add a new test package called `foo`.
+
+1. First create `kola/tests/foo/`
+2. Then `echo "package foo" > kola/tests/foo/foo.go`
+3. Next, edit `kola/registry/registry.go` and add this to the imports `_ "github.com/coreos/mantle/kola/tests/foo"`
+```golang
+package registry
+
+// Tests imported for registration side effects. These make up the OS test suite and is explicitly imported from the main package.
+import (
+        _ "github.com/coreos/mantle/kola/tests/coretest"
+        _ "github.com/coreos/mantle/kola/tests/crio"
+        _ "github.com/coreos/mantle/kola/tests/docker"
+        _ "github.com/coreos/mantle/kola/tests/etcd"
+        _ "github.com/coreos/mantle/kola/tests/foo"
+<snip/>
+```
+4. Lastly, use $EDITOR on `kola/tests/foo/foo.go` adding in new test groups and tests.
+
+
+
+
+
+# Full Example
```

Comment:
Should this be a `##` heading with the files being `###` so they nestle under the `Adding Tests` top level header?